plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/TestCassandraConnectorTest.java

```diff
@@ -1384,6 +1386,71 @@ public void testRowLevelDelete()
                 .hasStackTraceContaining("Delete without primary key or partition key is not supported");
     }
 
+    // test polymorphic table function
+
+    @Test
+    public void testNativeQuerySelectFromNation()
```

Comment:
Can we add additional test to ensure that `filterPushDown` doesn't happen with this PTF and also one with invalid column type.

Reply:
Added, but the remaining unsupported type in Cassandra connector is only time type and #15802 is going to support the type.

Reply:
Is there is any type in cassandra - which is difficult to support or will not be supported in future. Or maybe some user defined type.

Reply:
I don't there is. 25/26 types including UDT are supported.